db/migrate/20160127140802_add_digital_location_to_delivery_location.rb

```diff
@@ -0,0 +1,5 @@
+class AddDigitalLocationToDeliveryLocation < ActiveRecord::Migration
+  def change
+    add_column :locations_delivery_locations, :digital_location, :boolean
+  end
```

Comment:
If we just want a default, true is fine and then manually editing to get things into a correct state.

As I said in the original issue,

```ruby
Locations::DeliveryLocation.select { |d| d.pickup_location? }
```

gets us really close; only two ("Lewis Library, Geosciences and Map Library" and "ReCAP Reading Rooom") would be incorrect; so if getting more fancy seems appropriate, that might be an idea.